assets/runtime/config/gitlabhq/gitlab.yml

```diff
@@ -568,6 +568,19 @@ production: &base
     #   enabled: true
     #   host: localhost
     #   port: 3808
+    ## Monitoring
+  # Built in monitoring settings
+  monitoring:
```

Comment:
I don't think we can remove the whole key - I don't see this be done anywhere in the `.yml`